### ISSUE TITLE
InvalidateMeasure() does not trigger a native layout.

### ISSUE THREAD

**Issue by Clancey** [dotnet/maui#4102]
### Description

If you call ` label.Handler.NativeView.InvalidateMeasure(label)` it doesnt trigger a layout.

This is problematic, as we need to trigger re-layouts when font sizes change.

### Steps to Reproduce

I am using comet, My View Heirarchy is
`
VStack{
Text(),
Button()
}
`
When I set the text to exact sizing, it matches the initial value. If on button click, I extend the labels text to more characters, it doesn't relayout.

I also added
```
LabelHandler.LabelMapper.Add(nameof(IText.Text), (h, l) => {
				h.NativeView.InvalidateMeasure(l);
				LabelHandler.MapText(h, l);
			});
``` 

to force the invalidate

### Version with bug

Preview 11

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, Android, macOS

### Affected platform versions

All

**Comment by XamlTest** [comment]
Thank you so much for your bug report! Unfortunately, we cannot reproduce your issue with the information above, could you provide a sample project and screenshots or more details that we can reproduce your issue? That should greatly speed up the process, thanks!

**Comment by mattleibow** [comment]
I got a feeling you are supposed to use some new method from IView or something... @hartez is that still something as I see a explicit implementation or did that all change?

**Comment by VincentBu** [comment]
Will calling `InvalidateMeasure()` trigger the `UnLoaded` event? I write a sample project [MauiApp4102.zip](https://github.com/dotnet/maui/files/8649232/MauiApp4102.zip).
Please add breakpoint in `UnLoadedEvent` and run the project. Then click the "Change Font Size" button. 

I validated with vs main build(32507.29.main) and the `UnLoaded` event wasn't triggered.

**Comment by mattleibow** [comment]
> Will calling `InvalidateMeasure()` trigger the `UnLoaded` event?

No. InvalidateMeasure should cause the UI to be layout it's views.

**Comment by taublast** [comment]
Try `InvalidateMeasureNonVirtual(InvalidationTrigger.HorizontalOptionsChanged); `this will cause the parent view to layout and call your `MeasureOverride `and `ArrangeOverride`. Do not forget to set manually `DesiredSize `inside your `MeasureOverride ` in case you do not call its `base`.

**Comment by jinxinjuan** [comment]
Repro on version 17.6.0 Preview 3.0 with the above project [MauiApp4102.zip](https://github.com/dotnet/maui/files/8649232/MauiApp4102.zip). It doesn't cause the UI to be layout it's views.

**Comment by praveenkumarkarunanithi** [comment]
This issue has been validated in Visual Studio Code 1.111.0 with MAUI version 10.0.50. It cannot be reproduced on any platform.

https://github.com/user-attachments/assets/d56a88e1-48f0-4528-9241-d35fb5102242
